packages/@react-stately/tooltip/test/TooltipManager.test.js

```diff
@@ -0,0 +1,137 @@
+import {TooltipManager} from '../';
+
+describe('TooltipManager', () => {
```

Comment:
I'm assuming that the remainder of the tests will be added in https://jira.corp.adobe.com/browse/RSP-1471
Can you please make notes of places that you diverged from the tests outlined there vs here and why?

Reply:
That's a toast container ticket? 
I'll look at your outline again

Reply:
yup, whoops, copy pasta mistake https://jira.corp.adobe.com/browse/RSP-1367